app/design/frontend/default/models/template/user/table.phtml

```diff
@@ -0,0 +1,54 @@
+<form role="form" action="<?php echo $this->getUrl('user/index/save'); ?>" method="post">
+    <ul class="form-list">
+      <li>
+           <label for="firstname" class="required">FIRST NAME :</label>
```

Comment:
FIRST NAME dont write text like this. already commneted about this vihor pull request